services/repository-managers/src/test/java/org/sagebionetworks/repo/manager/file/FileHandleManagerImplTest.java

```diff
@@ -840,8 +825,7 @@ public void testGetRedirectURLForFileHandleGoogleCloudNoFileName() throws Datast
 		googleCloudFileHandle.setContentType("text/plain");
 
 		when(mockFileHandleDao.get(googleCloudFileHandle.getId())).thenReturn(googleCloudFileHandle);
-		String expectedURL = "https://google.com";
-		expectedURL += "&" + RESPONSE_CONTENT_TYPE + "=" + URLEncoder.encode(googleCloudFileHandle.getContentType(), "UTF-8");
+		String expectedURL = "https://google.com?response-content-type=text%2Fplain";
```

Comment:
we should also have a test for both content type and disposition is not available.